data/en/get_started.json

```diff
@@ -467,23 +467,23 @@
               "details": "content/en/get-started/cards/vm-435.md"
             },
             {
-              "id": "docker-621",
+              "id": "docker-622",
               "item": {
                 "title": "CDAP for Docker",
                 "description": "Download a standalone zip file for trying out CDAP on your Linux or Mac OSX machine.",
                 "img": "/images/get-started/docker.png",
                 "buttons": [
                   {
-                    "buttonTitle": "CDAP 6.2.1 for Docker",
-                    "id": "version-621-mac",
+                    "buttonTitle": "CDAP 6.2.2 for Docker",
+                    "id": "version-622-mac",
```

Comment:
Should we use docker-621 here?

Reply:
@supriyapatil-google For minor and patch versions we surface only the latest. In this case since its a patch version release we only show 6.2.2 and not 6.2.1 or 6.2.0. 

This way each major/minor version always shows the latest. Similar to this,
![Screen Shot 2020-09-28 at 8 54 13 PM](https://user-images.githubusercontent.com/1452845/94510931-d85f1c00-01cc-11eb-815c-8227a4708d7b.png)

Even though we have released 6.1.2, 6.1.3, 5.1.1 etc., we only surface the latest from them.

Reply:
I meant docker-622 here since this is for Docker and we use the docker builds in all other selections. Is there something different about this selection or the old setting was a typo?

Reply:
@supriyapatil-google Hhmm I am not entirely sure. We seem to be using the same `version-xxx-mac` for both vm and docker section buttons. I am not sure if we have google analytics setup that will track downloading docker and vm as desktop downloads?

Reply:
SG. Lets leave it as is. Thanks!